shell/browser/browser_mac.mm

```diff
@@ -325,6 +342,34 @@ bool CheckLoginItemStatus(bool* is_hidden) {
   return prevent_default;
 }
 
+// Modified from chrome/browser/ui/cocoa/l10n_util.mm.
+void Browser::ApplyForcedRTL() {
+  NSUserDefaults* defaults = NSUserDefaults.standardUserDefaults;
+
+  auto dir = base::i18n::GetForcedTextDirection();
+
+  // An Electron app should respect RTL behavior of application locale over
+  // system locale.
+  auto should_be_rtl =
+      dir == base::i18n::RIGHT_TO_LEFT || (IsAppRTL() && !IsSystemRTL());
+  auto should_be_ltr =
+      dir == base::i18n::LEFT_TO_RIGHT || (!IsAppRTL() && IsSystemRTL());
```

Comment:
Why do we care if the system is RTL given application locale takes priority, can't this be simplified to the following ?

```suggestion
  auto should_be_rtl =
      dir == base::i18n::RIGHT_TO_LEFT || IsAppRTL();
  auto should_be_ltr =
      dir == base::i18n::LEFT_TO_RIGHT || !IsAppRTL();
```

Reply:
fair point!